.github/workflows/integrate-and-deploy.yml

```diff
@@ -46,11 +46,18 @@ jobs:
       -
         name: Establish image name
         id: image
+        env:
```

Comment:
artifact from a previous age, can be removed

Reply:
Done in commit 8bd4169.